TextureViewtoSurfaceView/README.md

```diff
@@ -0,0 +1,45 @@
+
+TextureView to SurfaceView
+===================================
+
+This sample shows the similarities and differences between [TextureView][1] and [SurfaceView][2] 
+with the goal ultimately being to migrate away from TextureView entirely and migrating to using 
+SurfaceView for most, if not all cases.
+
+[1]: https://developer.android.com/reference/android/view/TextureView
+[2]: https://developer.android.com/reference/android/view/SurfaceView
+
+Introduction
+------------
+
+SurfaceView has been around since the beginning of Android (API 1). SurfaceView is more optimized 
+and gives the ability to display 10-Bit HDR content on supported devices. Other benefits include:
+
+* Better power efficiency
+* 10-Bit HDR Support (dependent on support from the display on the device)
```

Comment:
I'm not sure this bullet point is needed, since you already mention that there is 10-bit HDR support above